pandas/tests/test_strings.py

```diff
@@ -2220,27 +2220,31 @@ def test_center_ljust_rjust_fillchar(self):
             result = values.str.rjust(5, fillchar=1)
 
     def test_zfill(self):
```

Comment:
the issue mentions also affecting ``rjust``, ``pad``, ``center``, can you add / update tests as needed for those as well.